WcaOnRails/config/i18n-tasks.yml.erb

```diff
@@ -157,7 +157,7 @@ ignore:
 
   I18n::Tasks.add_scanner 'I18n::Tasks::Scanners::PatternMapper',
     only: %w(*.js),
-    patterns: [['I18nHTMLTranslate\s+i18nKey=%{key}', '%{key}']]
+    patterns: [['I18nHTMLTranslate\s+i18nKey=%{key}', '%{key}', 'tArray(%{key})']]
```

Comment:
I think you have mis-understood some of the pattern mapping used in `i18n-keys`.

The `patterns` array is of the shape `[[a,b],[a,b],[a,b]]` where `a` = "something that appears in our source code" and `b` = "the key that this library should extrapolate from `a`". This can be useful because normally, when checking for unused keys the library only considers occurences like `I18n.t(some_key)`. When we do `<I18nHTML key=some_key>`, we need to teach it that `some_key` should be extrapolated from that React tag.

Having an entry in the array in the shape of `[a,b,c]` like you're trying here won't work, I'm afraid.

Reply:
Or, to summarize: If `a` appears in our source code, pretend it's actually `b`.

Reply:
See also https://github.com/thewca/worldcubeassociation.org/pull/6939/files#diff-c00fd3fabe65500332a033bff4b9f9805b48889b2e8eca1764fa3c09a6e4586fL35-L39

The GitHub syntax highlighting is broken and marks the entire paragraph as comment (despite the fact that only the first line actually is a comment) but you can see that we used to be doing `I18n.t(foo.bar.#{index})` using string interpolation and "somehow" ™️ the library was able to figure out the (un)used keys.

Reply:
That makes sense, I was hoping that would work but I can see why not now :P

Thanks!

Reply:
Please revert the change then :)

Reply:
Whoops sure thing!